src/components/scenes/Staking/StakeOptionsScene.tsx

```diff
@@ -46,6 +47,10 @@ export const StakeOptionsScene = (props: Props) => {
     return currencyWallets[walletId]
   })
 
+  const pluginId = wallet?.currencyInfo.pluginId
+  const tokenId = pluginId ? getTokenId(account, pluginId, currencyCode) : undefined
+  const icon = React.useMemo(() => <CryptoIcon marginRem={[0, 0.5, 0, 0]} walletId={walletId} tokenId={tokenId} sizeRem={1.5} />, [])
```

Comment:
Missing walletId, and tokenId dependencies for the memo

Reply:
I'm also okay with inlining this icon JSX in the return like normal (CrypoIcon isn't an expensive function).

Reply:
This was actually copy/paste from line 29 but good catch. I'll move it inline